frontend/src/concepts/pipelines/content/createRun/RunPage.tsx

```diff
@@ -48,6 +49,14 @@ const RunPage: React.FC<RunPageProps> = ({ cloneRun, contextPath, testId }) => {
   const [cloneRunPipeline] = usePipelineById(cloneRunPipelineId);
   const [cloneRunExperiment] = useExperimentById(cloneRunExperimentId);
 
+  const isExperimentsAvailable = useIsAreaAvailable(SupportedArea.PIPELINE_EXPERIMENTS).status;
+
+  const sections = isExperimentsAvailable
+    ? Object.values(CreateRunPageSections)
+    : Object.values(CreateRunPageSections).filter(
```

Comment:
nit: Instead of a filter, requiring iteration, you could just omit experiment by means of spreading. E.g.

```
const allSections = Object.values(CreateRunPageSections);
const { [CreateRunPageSections.EXPERIMENT]: experiment, ...sectionsWithoutExperiment } = allSections;
```

then `sections={isExperimentsAvailable ? allSections : sectionsWithoutExperiment}`

Calling this a nitpick as I think this is temporary logic so won't consider blocking.

Reply:
yeah. one line is what i preferred. could go either way tho as the filter really is not that bad when going over 5 items.